js/scripts.js

```diff
@@ -0,0 +1,13 @@
+// scripts.js
+
+function getTriangleArea(a,h) {
+	if ((a <= 0) || (h <= 0)) {
+		return('Nieprawidłowe dane')
+	} else {
```

Comment:
zauważ, że `return` wychodzi z funkcji - co sprawia, że sam `else` jest mało potrzebny. Jak dane będą nieprawidłowe to trafimy na `return` stringa, natomiast jak nie trafimy na ten return, to znaczy, że dane są prawidłowe i możemy śmiało zwracać wynik.